main.rb

```diff
@@ -0,0 +1,108 @@
+# rubocop:disable all
+
+require './app'
+class Main
+    def initialize
+      @app = App.new
+      puts 'Welcome to School Library App!'
+    end
+  
+    def menu
+      puts %(
+  Please choose an option by enterin a number:
+  1 - List all books
+  2 - List all people
+  3 - Create a person
+  4 - Create a book
+  5 - Create a rental
+  6 - List all rentals for a given person id
+  7 - Exit)
+    end
+  
+    def create_person
+      print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+      person_type = gets.chomp
+      unless %w[1 2].include?(person_type)
+        puts 'Invalid option'
+        return
+      end
+      print 'Age: '
+      age = gets.chomp
+      print 'Name: '
+      name = gets.chomp
+      case person_type
+      when '1'
+        print 'Has parent permission? [Y/N]: '
+        @app.add_student(age, name, gets.chomp.downcase == 'y')
```

Comment:
- [ ] I applaud you greatly for implementing the previous changes, your app is greatly improved, however, there is still an issue when creating a student. I get the error in the image below, please look into it
<img width="911" alt="Screenshot 2023-07-01 at 17 11 33" src="https://github.com/Daniel-Kigozi/school_library/assets/64774936/1b96a8a6-2400-4c19-ae6f-93dbd75b923b">